tests/unit/test_provider.py

```diff
@@ -99,22 +99,26 @@ def setUp(self):
         self.harness.set_leader(True)
         self.harness.begin()
 
-    @patch("ops.testing._TestingPebbleClient.make_dir")
+    @patch("ops.testing._TestingPebbleClient.make_dir", Mock())
     @patch(
         "charms.loki_k8s.v0.loki_push_api.LokiPushApiProvider.unit_ip", new_callable=PropertyMock
     )
-    def test_relation_data(self, mock_unit_ip, *unused):
+    def test_relation_data(self, mock_unit_ip):
         mock_unit_ip.return_value = "10.1.2.3"
         expected_value = '{"url": "http://10.1.2.3:3100/loki/api/v1/push"}'
         self.assertEqual(expected_value, self.harness.charm.loki_provider._loki_push_api)
 
-    @patch("ops.testing._TestingPebbleClient.make_dir")
-    @patch("charms.loki_k8s.v0.loki_push_api.LokiPushApiProvider._generate_alert_rules_files")
-    @patch("charms.loki_k8s.v0.loki_push_api.LokiPushApiProvider._remove_alert_rules_files")
+    @patch("ops.testing._TestingPebbleClient.make_dir", Mock())
+    @patch(
+        "charms.loki_k8s.v0.loki_push_api.LokiPushApiProvider._generate_alert_rules_files", Mock()
+    )
+    @patch(
+        "charms.loki_k8s.v0.loki_push_api.LokiPushApiProvider._remove_alert_rules_files", Mock()
+    )
```

Comment:
Can the test be updated to work without these mocks?

Reply:
No